resources/yki/queries.sql

```diff
@@ -556,10 +556,6 @@ WHERE id = (SELECT re.id
 AND EXISTS (SELECT id
             FROM exam_session
             WHERE id = :exam_session_id
-              AND max_participants > (SELECT COUNT(1)
-                                      FROM registration
-              	                      WHERE exam_session_id = :exam_session_id
-	                                    AND state IN ('COMPLETED', 'SUBMITTED', 'STARTED'))
```

Comment:
Tämä taitaa nyt löysätä rajoitteita vähän suunniteltua enemmän. Tuo `AND state IN ('COMPLETED', 'SUBMITTED', 'STARTED')` pitäisi varmaan ottaa mukaan esim. omaksi ehdoksi riville 556.